src/Consolonia.Themes/Fluent/FluentTheme.axaml

```diff
@@ -42,12 +42,19 @@
             <x:String x:Key="ThemeCheckedSymbol">✓</x:String>
             <x:String x:Key="ThemeCloseSymbol">✕</x:String>
 
-            <x:String x:Key="ThemeToggleSwitchOffSymbol">◯</x:String>
-            <x:String x:Key="ThemeToggleSwitchOnSymbol">⬤</x:String>
+            <x:String x:Key="ThemeToggleSwitchOffSymbol">●</x:String>
```

Comment:
I hate losing these theme characters. same comment as above?

Reply:
I just tried to match Avalonia:
![image](https://github.com/user-attachments/assets/247f6fa3-887b-48f7-a510-4e26ebb5958e)
it's filled black when off and filled white when on.
Should I revert?

Reply:
Also moved to separate PR: https://github.com/jinek/Consolonia/pull/324